app/assets/javascripts/components/form_label.jsx

```diff
@@ -0,0 +1,5 @@
+class FormLabel extends React.Component {
+  render() {
+    return <label htmlFor={this.props.for} key={this.props.for}> {this.props.text} </label>
```

Comment:
Can we get rid of the extra space inside the tag? Worried it might cause styling issues.